eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -0,0 +1,214 @@
+                        displayName: 'Publish ${{artifact.name}} package to NuGet.org'
+                        inputs:
+                          command: push
+                          packagesToPush: '$(Pipeline.Workspace)/staging/**/*.nupkg;!$(Pipeline.Workspace)/staging/**/*.symbols.nupkg'
+                          nuGetFeedType: external
+                          publishFeedCredentials: Nuget.org
+
+          - ${{if ne(artifact.options.skipSymbolsUpload, 'true')}}:
+            - deployment: UploadSymbols
+              displayName: Upload Symbols to Symbols Server
+              condition: and(succeeded(), ne(variables['Skip.SymbolsUpload'], 'true'))
+              environment: nuget
+              dependsOn: PublishPackage
+
+              pool:
+                vmImage: windows-2019
+
+              strategy:
+                runOnce:
+                  deploy:
+                    steps:
+                      - checkout: none
+                      - template: tools/clone-buildtools/clone-buildtools.yml@azure-sdk-build-tools
+                      - pwsh: |
+                          New-Item -Type Directory -Name staging -Path $(Pipeline.Workspace)
+                          Copy-Item $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed/${{artifact.name}}.[0-9]*.[0-9]*.[0-9]* $(Pipeline.Workspace)/staging
+                          Get-ChildItem $(Pipeline.Workspace)/staging
+                        displayName: Copying ${{artifact.name}} to staging directory
+                      - task: MSBuild@1
+                        displayName: 'Upload Symbols for ${{artifact.name}}'
+                        inputs:
+                          solution: '$(AzureSDKBuildToolsPath)/tools/symboltool/SymbolUploader.proj'
+                          msbuildArguments: '/p:PackagesPath=$(Pipeline.Workspace)/staging /p:MSPublicSymbolsPAT=$(azuresdk-microsoftpublicsymbols-devops-pat) /p:MSSymbolsPAT=$(azuresdk-microsoft-devops-pat) /p:AzureSDKSymbolsPAT=$(azuresdk-azure-sdk-devops-pat)'
+
+          - ${{if ne(artifact.options.skipPublishDocs, 'true')}}:
+            - deployment: PublishDocs
+              displayName: Publish Docs to GitHub pages
+              condition: and(succeeded(), ne(variables['Skip.PublishDocs'], 'true'))
+              environment: githubio
+              dependsOn: PublishPackage
+
+              pool:
+                vmImage: windows-2019
+
+              strategy:
+                runOnce:
+                  deploy:
+                    steps:
+                      - checkout: azure-sdk-tools
+                      - pwsh: |
+                          New-Item -Type Directory -Name ${{artifact.safeName}} -Path $(Pipeline.Workspace)
+                          New-Item -Type Directory -Name packages -Path $(Pipeline.Workspace)/${{artifact.safeName}}
+                          New-Item -Type Directory -Name Docs.${{artifact.name}} -Path $(Pipeline.Workspace)/${{artifact.safeName}}
```

Comment:
As part of the other issue of putting each artifact in its own folder we should probably put the docs in that same folder which should simplify this step as well.

Reply:
Included in this [Issue](https://github.com/Azure/azure-sdk-tools/issues/435)